server/common/src/main/java/org/eclipse/lsp/cobol/common/copybook/PredefinedCopybookStore.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.common.copybook;
```

Comment:
Had to add these in `common` module , so that it could be used in `test` module. `test` module is not dependent on engine module

Reply:
same goes for `io` package as well